mitiq/interface/conversions.py

```diff
@@ -259,27 +259,28 @@ def qprogram_modifier(
 
 
 def accept_qprogram_and_validate(
-    cirq_circuit_modifier: Callable[..., Any]
+    cirq_circuit_modifier: Callable[..., Any],
+    one_to_many: Optional[bool] = False,
 ) -> Callable[..., Any]:
     """This decorator performs two functions:
 
-        1. Transforms a function of signature (cirq.Circuit -> Any) to
-        (QPROGRAM -> Any)
+        1. Transforms a function of signature (cirq.Circuit -> cirq.Circuit)
+        to (QPROGRAM -> QPROGRAM).
         2. Validates the incoming QPROGRAM instance to ensure Mitiq's error
         mitigation techniques can be applied to it.
 
     Args:
-        cirq_circuit_modifier: The function to scale.
+        cirq_circuit_modifier: The function modifying a Cirq circuit.
+        one_to_many: If True, cirq_circuit_modifier is expected to return
+            a sequence of Cirq circuits instead of a single Cirq circuit.
```

Comment:
```suggestion
        cirq_circuit_modifier: The function modifying a Cirq circuit.
        one_to_many: If True, ``cirq_circuit_modifier`` is expected to return
            a sequence of Cirq circuits instead of a single Cirq circuit.
```
**Warning**: this may make `flake8` unhappy.